core/lib/mno_enterprise/concerns/models/app_instance.rb

```diff
@@ -9,6 +9,7 @@ module MnoEnterprise::Concerns::Models::AppInstance
   included do
     property :created_at, type: :time
     property :updated_at, type: :time
+    property :app_id, type: :string
```

Comment:
I thought we were getting rid of this in #560

Reply:
we are, but little by little.